src/test/java/com/wikia/webdriver/testcases/templateclassification/Editing.java

```diff
@@ -0,0 +1,45 @@
+package com.wikia.webdriver.testcases.templateclassification;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.common.Navigate;
+import com.wikia.webdriver.elements.oasis.templateclassificiation.EntryPoint;
+import com.wikia.webdriver.elements.oasis.templateclassificiation.TemplateModal;
+
+import org.testng.annotations.Test;
+
+@Execute(asUser = User.STAFF, onWikia = "aga")
+public class Editing extends NewTestTemplate {
+
+  private Navigate navigate;
+  private EntryPoint entryPoint;
+  private TemplateModal templateModal;
+
+  private void init() {
+    this.navigate = new Navigate(driver);
+    this.entryPoint = new EntryPoint(driver);
+    this.templateModal = new TemplateModal(driver);
+  }
+
+  @Test()
+  public void Editing_1_open_and_close() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)